test/bats/test.bats

```diff
@@ -23,6 +24,7 @@ SLEEP_TIME=1
 }
 
 @test "validate crd add, replace and delete" {     
```

Comment:
It would be good to move this a local registry and possibly run this with something like Kind.

Reply:
Right, we have an issue to address it: https://github.com/deislabs/ratify/issues/284. It's part of e2e testing framework set up.